Flaky test: "when using language hook handler and spawning the server and handler is killed before execution should return with status 1"

Another annoying flaky little fellow in the Windows test suite:
```
1) CLI when called with arguments when using language hook handler and spawning the server and handler is killed before execution should return with status 1:
      AssertionError: expected 7 to equal 1
      + expected - actual
      -7
      +1
      
      at Context.it (C:\projects\dredd\test\integration\cli\cli-test.js:166:56)
      at callFn (C:\projects\dredd\node_modules\mocha\lib\runnable.js:334:21)
      at Test.Runnable.run (C:\projects\dredd\node_modules\mocha\lib\runnable.js:327:7)
      at Runner.runTest (C:\projects\dredd\node_modules\mocha\lib\runner.js:428:10)
      at C:\projects\dredd\node_modules\mocha\lib\runner.js:534:12
      at next (C:\projects\dredd\node_modules\mocha\lib\runner.js:348:14)
      at C:\projects\dredd\node_modules\mocha\lib\runner.js:358:7
      at next (C:\projects\dredd\node_modules\mocha\lib\runner.js:284:14)
      at Immediate.<anonymous> (C:\projects\dredd\node_modules\mocha\lib\runner.js:326:5)
```

--- a/src/dredd-command.js
+++ b/src/dredd-command.js
@@ -254,6 +254,11 @@ ${packageData.name} v${packageData.version} \
         this._processExit(1);
       });
 
+      // Fixes https://github.com/apiaryio/dredd/issues/1030
+      process.stdin.on('error', (err) => {
+        logger.error(`Error in stdin: ${err}`);
+      });
+
       // Ensure server is not running when dredd exits prematurely somewhere
       process.on('beforeExit', () => {
         if (this.serverProcess && !this.serverProcess.terminated) {

--- a/src/dredd-command.js
+++ b/src/dredd-command.js
@@ -31,6 +31,11 @@ class DreddCommand {
     if (!this.custom.argv || !Array.isArray(this.custom.argv)) {
       this.custom.argv = [];
     }
+
+    // Fixes https://github.com/apiaryio/dredd/issues/1030
+    process.stdin.on('error', (err) => {
+      logger.error(`Error in stdin: ${err}`);
+    });
   }
 
   setOptimistArgv() {
@@ -252,11 +257,6 @@ ${packageData.name} v${packageData.version} \
       this.serverProcess.on('error', (err) => {
         logger.error('Command to start backend server process failed, exiting Dredd', err);
         this._processExit(1);
-      });
-
-      // Fixes https://github.com/apiaryio/dredd/issues/1030
-      process.stdin.on('error', (err) => {
-        logger.error(`Error in stdin: ${err}`);
       });
 
       // Ensure server is not running when dredd exits prematurely somewhere

--- a/test/integration/cli/cli-test.js
+++ b/test/integration/cli/cli-test.js
@@ -163,7 +163,7 @@ describe('CLI', () => {
 
         after(done => killAll('test/fixtures/scripts/', done));
 
-        xit('should return with status 1', () => assert.equal(runtimeInfo.dredd.exitStatus, 1));
+        it('should return with status 1', () => assert.equal(runtimeInfo.dredd.exitStatus, 1));
 
         it('should return message announcing the fact', () => {
           if (process.platform === 'win32') {

